image/handler.go

```diff
@@ -0,0 +1,37 @@
+package image
+
+import (
+	"github.com/buildpacks/imgutil"
+	"github.com/docker/docker/client"
+	"github.com/google/go-containerregistry/pkg/authn"
+)
+
+type Handler interface {
+	InitImage(imageRef string) (imgutil.Image, error)
+	Docker() bool
+	Layout() bool
```

Comment:
Calling code might read better if we make a `Kind()` method instead that returns a constant (e.g., `docker`, `layout`, or `remote`)

Reply:
Done.